src/test/java/com/scottlogic/FilterTests/AuthorFilter.java

```diff
@@ -0,0 +1,32 @@
+package com.scottlogic.FilterTests;
+
+import com.scottlogic.UserPost;
+import com.scottlogic.context.FilterContext;
+import com.scottlogic.filter.FilterAuthor;
+import com.scottlogic.utils.UserPostFilterData;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+public class AuthorFilter {
+
+    FilterContext filterAuthor = new FilterContext(new FilterAuthor());
```

Comment:
What is the subject of this unit test?
If you want to test FilterAuthor, then you should call this FilterAuthorTest and use the class directly.

No need for a wrapper like your FilterContext. That's like having an ATM with a name filter, but to unit test the name filter you've created an ATM for it. Who knows what the ATM would do that might affect the tests.

Reply:
Fixed